clients/python/Makefile

```diff
@@ -1,7 +1,7 @@
 all: install tidy
 
 IMG_VERSION ?= latest
-IMG ?= ghcr.io/kubeflow/model-registry/server:${IMG_VERSION}
+IMG ?= ghcr.io/kubeflow/model-registry/server
```

Comment:
If this remained the same i would hit an error where the `make image/build` would try and create an image called `ghcr.io/kubeflow/model-registry/server:latest:lastest` and that would cause a failure.